flytekit/extras/tensorflow/tensor/tensor_transformer.py

```diff
@@ -0,0 +1,82 @@
+import pathlib
```

Comment:
@VPraharsha03, can you move this file to tensor.py? Let's not have a tensor folder, instead let's have a `tensor.py` file under `flytekit/extras/tensorflow`.